tutorindigo/plugin.py

```diff
@@ -193,6 +192,18 @@ def _override_openedx_docker_image(
 for filename in javascript_files:
     if filename in PIPELINE['JAVASCRIPT']:
         PIPELINE['JAVASCRIPT'][filename]['source_filenames'] += dark_theme_filepath
+
+{% if INDIGO_THEME_COOKIE_NAME %}
+MFE_CONFIG['THEME_COOKIE_NAME'] = '{{ INDIGO_THEME_COOKIE_NAME }}'
+{% endif %}
+""",
```

Comment:
remove this

Reply:
If we remove this, then how will we inform MFEs that the dark toggle button should be enabled or not?

Reply:
I don't understand why this should be necessary. The templates will not include the toggle part thanks to the `{% if INDIGO_ENABLE_DARK_TOGGLE %}` conditions. The condition will be applied at build-time, making the run-time condition unnecessary. Did you try to enable/disable the feature without modifying `MFE_CONFIG`?

Reply:
I got your point. I've removed the build-time conditions for `settings` patches in this [commit.](https://github.com/overhangio/tutor-indigo/pull/97/commits/3e9ac7f00a071ad4b035a8f09e0af531ea0e549d)

Could you please review the code again when available? Also, are we going to make it a part of Sumac (and merge in nightly instead of master) as it's a separate feature?

Reply:
:+1: